tap-setup-scripts/src/functions.sh

```
@@ -698,17 +699,12 @@ function tapPrepIterateClusterToken {
   aws eks update-kubeconfig --name ${MY_CLUSTER_NAME}
 
   ITERATE_CLUSTER_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
-  ITERATE_CLUSTER_SECRET=$(kubectl -n tap-gui get sa tap-gui-viewer -o=json  | jq -r '.secrets[0].name')
-  ITERATE_CLUSTER_TOKEN=$(kubectl -n tap-gui get secret ${ITERATE_CLUSTER_SECRET} -o=json \
+  ITERATE_CLUSTER_TOKEN=$(kubectl -n tap-gui get secret tap-gui-viewer -o=json \
```

*hint*: This could also be done with `kubectl` only, with a crisp `kubectl -n tap-gui get secret tap-gui-viewer -o go-template='{{ .data.token | base64decode }}'`

---

Thanks for the info. I guess we have used jq mostly.  Prefer not to change now.

---

*non-blocking*

If feasible, I think no pipeline should still be preferred as I don't think we run with `pipefail` nor do we run `jq` with `-e`, and thus just swallows a lot of error cases.